cmake/Modules/HandleSanitizer.cmake

```diff
@@ -88,8 +88,11 @@ if (GALOIS_USE_SANITIZE_COVERAGE)
 endif()
 
 if (GALOIS_USE_SANITIZER MATCHES ".*Undefined.*")
-  set(BLACKLIST_FILE "${CMAKE_SOURCE_DIR}/config/sanitizers/ubsan_blacklist.txt")
-  if (EXISTS "${BLACKLIST_FILE}")
+  message("CURRENT SOURCE DIR: ${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Is this debugging code? Probably delete. It doesn't seem that helpful to others in its current state.

Reply:
Thanks. Yes, this is just debugging code. I'll remove it.